assets/styles/common/_variables.scss

```diff
@@ -94,7 +94,7 @@ $white: #fff;
 $gray: #231f20;
 $dark-gray: #65676b;
 $medium-gray: #b0b4bb;
-$medium-gray-quotes: #65676b;
+$medium-gray-quotes: #8c9097;
```

Comment:
Are you sure? This variable is used across the web for multiple other purposes. Changing it will change color on things that are not realted to the task.